libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -205,7 +205,10 @@ const setScaleBounds = produce((layers: ManagedDataLayer[]) => {
 });
 
 function computeBounds(axis: 'x' | 'y', padding: number, datasets: Dataset[], annotations?: ChartAnnotations): Partial<NumberRange> {
-  const values = datasets.flatMap((dataset) => dataset.data.map((point) => point[axis])).filter(isNotNaN);
+  const values = datasets
+    .flatMap((dataset) => dataset.data.map((point) => point[axis]))
+    .flat()
+    .filter(isNotNaN);
```

Comment:
I added `.flat()` because the x-value for medications can now be a `[number, number]` tuple